tests/src/main/java/net/neoforged/neoforge/debug/client/ClientEventTests.java

```diff
@@ -65,12 +66,12 @@ static void clientPlayerChangeGameTypeEvent(final ClientPlayerChangeGameTypeEven
         test.requestConfirmation(Minecraft.getInstance().player, Component.literal("Did you just change your game mode from " + event.getCurrentGameType() + " to " + event.getNewGameType() + "?"));
     }
 
-    @TestHolder(description = { "Tests if the RegisterRenderBuffersEvent event is fired and whether the registered render buffer is represented within a fixed render buffer map" }, enabledByDefault = true)
+    @TestHolder(description = { "Tests if the RegisterRenderBuffersEvent event is fired" }, enabledByDefault = true)
```

Comment:
The description should be reverted since the check for the removed part was added back.